spring-cloud-stream-binders/spring-cloud-stream-binder-redis/pom.xml

```diff
@@ -44,6 +44,11 @@
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-cloudfoundry-connector</artifactId>
```

Comment:
Agreed. Having this explicit in each binder that is supported by cloud-connector/service-binding is fine. That said, we should go ahead and update the rabbit binder POM as we're doing this for redis.